### PR TITLE
Closes #16903: Update release process to use Transifex CLI client

### LINKED ISSUE #16903: Update documented process for initializing a PR from Transifex
### Change Type

Correction

### Area

Development

### Proposed Changes

The process for initiating a pull request from Transifex involves:

1. Changing the configured threshold from 100% proofread to 100% translated
2. Waiting for the PR to be opened & completed
3. Merging the PR
4. Resetting the threshold to 100% proofread to avoid PR noise

### PR DESCRIPTION
**Pull request by jeremystretch**
### Fixes: #16903

- Introduce a Transifex configuration file
- Update the release & translation docs with instructions for running the Transifex CLI client
- Remove obsolete screenshots

### CODE REVIEW

File: docs/development/translations.md [hunk 1]
```diff
@@ -16,26 +16,31 @@ To update the English `.po` file from which all translations are derived, use th
...
 Typically, translated strings need to be updated only as part of the NetBox [release process](./release-checklist.md).
 
 Check the Transifex dashboard for languages that are not marked _ready for use_, being sure to click _Show all languages_ if it appears at the bottom of the list. Use machine translation to round out any not-ready languages. It's not necessary to review the machine translation immediately as the translation teams will handle that aspect; the goal at this stage is to get translations included in the Transifex pull request.
 
-To update translated strings, start by initiating a sync from Transifex. From the Transifex dashboard, navigate to Settings > Integrations > GitHub > Manage, and click the **Manual Sync** button at top right.
+To download translated strings automatically, you'll need to:
 
-![Transifex manual sync](../media/development/transifex_sync.png)
+1. Install the [Transifex CLI client](https://github.com/transifex/cli)
+2. Generate a [Transifex API token](https://app.transifex.com/user/settings/api/)
```
Comment (arthanson) [inline docs/development/translations.md:31]: I think should add setting TX_TOKEN in environment file here.

**Review by arthanson — CHANGES_REQUESTED** [review]
Running the curl command for installing the transifex/cli will create two files in root LICENSE, TX, do we want to add those to gitignore?

**Comment by jeremystretch** [comment]
The `tx` client should be installed somewhere outside the repo (e.g. `/usr/bin/` or similar).

File: docs/development/translations.md (hunk 1 above)
Comment (jeremystretch) [reply docs/development/translations.md:31]: I added it to the `tx pull` command.

**Comment by jeremystretch** [comment]
CI Failure is unrelated (see #17950).

**Pull request merged**